CMakeLists.txt

```diff
@@ -210,11 +210,19 @@ endif()
 add_subdirectory(src bin)
 
 # Functional Tests
-if(BUILD_FUNCTIONAL_TESTS)
+if(BUILD_FUNCTIONAL_TESTS OR BUILD_EXAMPLES)
```

Comment:
Could you please set the default value for example_build option explicitly in the init section here https://github.com/mcao59/oneMKL/blob/6d8f9720a0221ab76d9234b98e2706308024fe93/CMakeLists.txt#L59  ?

Reply:
Added